class/legacy-wallet.js

```diff
@@ -131,9 +126,7 @@ export class LegacyWallet extends AbstractWallet {
       const txids = await BlueElectrum.getTransactionsByAddress(this.getAddress());
 
       await this.setTransactions(txids);
-    } catch (Err) {
-      console.warn(Err.message);
-    }
+    } catch (Err) {}
```

Comment:
I spotted that in many places we're catching errors and do nothing then. Why don't we want to log it out as a warn or something?

Reply:
I would rather suggest adding Sentry.addBreadcrumb in there instead of removing the catch at all.

Reply:
done